public/admin/config.yml

```diff
@@ -567,25 +524,138 @@ collections:
       - *image_config
       - name: category
         label: Category
-        widget: relation
-        collection: "ecosystem_projects_categories"
-        value_field: "slug"
-        search_fields:
-          - name
-        display_fields:
-          - name
+        widget: select
+        options:
+          - label: "Artificial Intelligence"
+            value: "artificial-intelligence"
+            subcategories: ["ai-productivity-utilities"]
+          - label: "Education, Academia & Health"
+            value: "education-academia-health"
+            subcategories: ["health"]
+          - label: "Finance"
+            value: "finance"
+            subcategories: ["leasing"]
+          - label: "Media & Entertainment"
+            value: "media-entertainment"
+            subcategories: ["arts-collectibles-nfts", "photo-video"]
+          - label: "Network"
+            value: "network"
+            subcategories: ["network-tools"]
+          - label: "Public Goods & DWeb"
+            value: "public-goods-dweb"
+            subcategories:
+              ["dweb", "governance-daos-public-goods", "social-impact"]
+          - label: "Science, Climate & Space"
+            value: "science-climate-space"
+            subcategories: ["climate", "space", "decentralized-science"]
+          - label: "Storage"
+            value: "storage"
+            subcategories:
+              [
+                "data-storage-management",
+                "storage-provider",
+                "storage-services",
+                "storage-software",
+                "infrastructure",
+              ]
+          - label: "Tooling & Productivity"
+            value: "tooling-productivity"
+            subcategories:
+              [
+                "data-retrieval",
+                "developer-tools",
+                "dapp",
+                "wallets-identity-authentication",
+                "bridges-oracles",
+              ]
       - name: subcategory
         label: Subcategory
-        widget: relation
-        collection: "ecosystem_projects_subcategories"
-        value_field: "slug"
-        search_fields:
-          - name
-        display_fields:
-          - name
-        filter:
-          field: "parent_category"
-          value: "{{fields.category}}"
+        widget: select
+        options:
+          # Artificial Intelligence subcategories
+          - label: "AI Productivity & Utilities"
+            value: "ai-productivity-utilities"
+            parent_category: "artificial-intelligence"
```

Comment:
What is the `parent_category` field for? These should be removed in this PR, right?